abs.yaml

```diff
@@ -0,0 +1,2 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/astropy-iers-data-feedstock/pr3/3897050
```

Comment:
Could you enable `py313` support if it's possible? 
`scikit-image 0.25.0` requires it in tests, see https://github.com/AnacondaRecipes/scikit-image-feedstock/pull/22/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR113

Reply:
Done!